Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+
+
+namespace BootCamp.Chapter
+{    public class Lesson3
+    {   public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Clear();
+                GatherData(out string firstName, out string lastName, out int age, out float weight, out float height);
+                DisplayOutput(firstName, lastName, age, weight, height);
+            }
+        }
+        public static string GetStringInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string value = Console.ReadLine();
+            
+            if (Empty(value))
+            {
+                Console.Write("Name cannot be empty.");
+                return "-";
+            }
+
+            //default case
+            return value;
+        }
+        public static int GetIntInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string value = Console.ReadLine();
+
+            if (Empty(value))
+                return 0;
+
+            if (int.TryParse(value, out int parsedInt))
+            {
+                return parsedInt;
+            }
+
+            //default case
+            BadInput(value);
+            return -1;
+            
+        }
+        public static float GetFloatInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string value = Console.ReadLine();
+
+            if (Empty(value))
+            {
+                return 0;
+            }
+
+            if (float.TryParse(value, out float parsedfloat))
+            {
+                return parsedfloat;
+            }
+
+            //default case
+            BadInput(value);
+            return -1;
+
+        }
+        // Input function
+        public static void GatherData(out string firstName, out string lastName, out int age, out float weight, out float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider refactoring this method in order to remove the need for this 'out' modifier.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886326)